MatterControlLib/CustomWidgets/XyCalibrationSelectPage.cs

```diff
@@ -188,7 +188,9 @@ await printer.Bed.LoadContent(new EditContext()
 					ContentStore = null // No content store for GCode
 				});
 
-				await printer.Connection.StartPrint(finalGCodePath, calibrationPrint: true);
+				// TODO: Reimplement
+				// await printer.Connection.StartPrint(finalGCodePath, calibrationPrint: true);
```

Comment:
Possibly no follow-up required - consider if TODO can be removed

Reply:
Issue created - MatterHackers/MCCentral#5762
Consider making StartPrint async again